bucoffea/scripts/run_quick.py

```diff
@@ -2,12 +2,21 @@
 
 from bucoffea.helpers.dataset import extract_year
 from bucoffea.processor.executor import run_uproot_job_nanoaod
-from bucoffea.monojet import monojetProcessor
 from bucoffea.helpers.cutflow import print_cutflow
 from coffea.util import save
 import coffea.processor as processor
+import argparse
+
+def getDatasetType():
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('type', type=str, help='The processor to be run. (monojet or vbfhinv)')
```

Comment:
Change `type` to something less generic (e.g. `processor`) so that it is more clear what the argument does